### PR TITLE
More mindbreaker effects

### PR DESCRIPTION
**Pull request by unid15**
chance of

-welder sounds
-screwdriver sounds
-You feel a tiny prick (why wasn't this in already :alien:)
-a fake emergency shuttle arrival message
-seeing a flashbang

### CODE REVIEW

File: code/modules/flufftext/Hallucination.dm [hunk 1]
```diff
@@ -143,6 +143,22 @@ mob/living/carbon/proc/handle_hallucinations()
 								'sound/hallucinations/look_up1.ogg', 'sound/hallucinations/look_up2.ogg', 'sound/hallucinations/over_here1.ogg', 'sound/hallucinations/over_here2.ogg', 'sound/hallucinations/over_here3.ogg',\
 								'sound/hallucinations/turn_around1.ogg', 'sound/hallucinations/turn_around2.ogg', 'sound/hallucinations/veryfar_noise.ogg', 'sound/hallucinations/wail.ogg')
 							src << pick(creepyasssounds)
+						if(13)
+							if(rand(0,1)==0)
```
Comment (ComicIronic) [inline code/modules/flufftext/Hallucination.dm]: if(prob(50))

File: code/modules/flufftext/Hallucination.dm [hunk 2]
```diff
@@ -143,6 +143,22 @@ mob/living/carbon/proc/handle_hallucinations()
...
+								src << 'sound/items/Welder.ogg'
+							else
+								src << 'sound/items/Welder2.ogg'
+						if(14)
+							if(rand(0,1)==0)
```
Comment (ComicIronic) [inline code/modules/flufftext/Hallucination.dm]: Seriously, why would you do this
if(prob(50))

**Comment by 9600bauds** [comment]
hell yes
Add a flashbang to the tiles that can appear, too

**Comment by 9600bauds** [comment]
Move the shuttle cal effect to the switch(halpick)

**Comment by 9600bauds** [comment]
:+1:

**Comment by mrpain666** [comment]
YES!

AND MALF AI SOUNDS PLEASE!

OR LIKE, NUKE ARMING SOUNDS!

OH FUCK YES.

**Comment by SonixApache** [comment]
@mrpain666 fake code delta warnings would be amazing

**Pull request merged**